sdk/storage/Azure.Storage.Common/api/Azure.Storage.Common.netstandard2.0.cs

```diff
@@ -47,7 +47,9 @@ public partial struct StorageTransferOptions : System.IEquatable<Azure.Storage.S
     public partial class StorageTransferStatus
     {
         public StorageTransferStatus() { }
-        public long bytesTransferred { get { throw null; } set { } }
+        public long bytesTransferred { get { throw null; } }
+        public long failedTransfers { get { throw null; } }
+        public long skippedTransfers { get { throw null; } }
         public long successfulTransfers { get { throw null; } }
```

Comment:
nit: props should start with capital letter.

Reply:
Done.